README.md

```diff
@@ -217,7 +228,7 @@ Topic information: Topic: /chatter | Type: std_msgs/String | Count: 9 | Serializ
                    Topic: /my_chatter | Type: std_msgs/String | Count: 18 | Serialization Format: cdr
 ```
 
-### Converting bags
+### Converting bags (merge, split, etc.)
```

Comment:
The link to this section in the list of verbs under _Using rosbag2_ needs to be updated. Search for `#converting-bags`.

Reply:
Added `<a id="heading-id"></a>` for all the anchor links to allow for consistent id while allowing for changing the heading title.